PhpAmqpLib/Channel/AMQPChannel.php

```diff
@@ -844,13 +844,19 @@ public function basic_nack($delivery_tag, $multiple = false, $requeue = false)
      *
      * @param string $consumer_tag
      * @param bool $nowait
+     * @param bool $noreturn
      * @return mixed
      */
-    public function basic_cancel($consumer_tag, $nowait = false)
+    public function basic_cancel($consumer_tag, $nowait = false, $noreturn = false)
```

Comment:
when sending a `basic.cancel` the server can still send messages to the consumer until it receives the actual `basic.cancel`. Wouldn't this allow for a race condition where messages could be lost?